qualang_tools/voltage_gates/README.md

```diff
@@ -0,0 +1,202 @@
+# VoltageGateSequence
+
+The `VoltageGateSequence` class facilitates the creation and management of complex pulse sequences for quantum operations, allowing for dynamic voltage control, ramping, and compensation across multiple gate elements.
+
+> **Note**: Make sure to initialize the class before opening the `qm`!
+
+> **Note**: Make sure the check the timing of the pulses in simulation or use `strict_timing()` to avoid real-time gaps!
+
+## Features
+
+- **Dynamic Voltage Control**: Define voltage levels (points) in gate-space and durations for multiple elements.
+- **Step and Ramp Support**: Add steps and ramps to the pulse sequence.
+- **Compensation Pulse**: Keeps track of average voltage levels for compensation pulses.
+- **Configuration Management**: Updates the provided configuration with necessary operations and waveforms for the sequence.
+
+## Initialization
+
+To use the `VoltageGateSequence` class, start by initializing it with the necessary configuration and elements.
+
+```python
+from voltage_gate_sequence import VoltageGateSequence
+
+# Configuration dictionary for OPX
+configuration = {
+    "elements": {...},
+    "pulses": {...},
+    "waveforms": {...}
+}
+
+# List of elements involved in the gate operations
+elements = ["P5_sticky", "P6_sticky"]
+
+# Initialize the VoltageGateSequence
+seq = VoltageGateSequence(configuration, elements)
+```
+
+## Methods
+
+### `add_step`
```

Comment:
I would show the `add_points` function before the `add_step` to illustrate what voltage_point_name means and how to set it up. I think that adding a simple charge stability map picture with relevant points would be helpful to illustrate the purpose, what do you think?

Reply:
Yes, good idea

Reply:
Done